README.md

```diff
@@ -7,90 +7,278 @@
+The cheapest option is most likely to buy compatible new parts for our existing
+computer. However, we've never attempted tinkering with hardware in our computer
+before, and after buying all the tools and parts, we follow an instructional
+video to make the upgrades. In this case, we only have one computer to work on,
+and nowhere to test out opening up a computer and connecting new parts. We have
+to be careful not to damage anything else that makes the computer work.
 
-Lukcily, with code we have a "sandbox", or a safe place, to experiment with different outcomes. With `git` and GitHub, we're given "branching" as an option so that we can _branch out_ from our original repo and any working code it contains, and make changes separately. If the code works out, we can keep it! If not, we can throw it out and start fresh. What a relief!
+It would be nice if we could test out our computer upgrading skills first.
+Luckily, we _do_ have this option when coding!
 
 ## Explain the Primary Role of Branching
 
-Play off of this reference:
-"Wouldn't it be great if you could try things out in "sandbox" and then throw it away, with no consequence, if it didn't work out? "Branches" in git allow exactly this. You can propose "parallel universes" of code that can be compared, discussed, tried out, and ultimately imported or thrown away."
+If we had an identical computer that we could experiment, we could worry less
+about making any mistakes that could impact our working machine. This is the
+exact concept of branching.
 
-<Go into a more specific real world applications of branching> 
+With code we have a "sandbox", or a safe place, to experiment with different
+outcomes. With `git` and GitHub, we're given "branching" as an option so that we
+can _branch out_ from our original repo and any working code it contains, and
+make changes separately. If the code works out, we can keep it! If not, we can
+throw it out and start fresh. What a relief!
+
+Branches can be merged into any one branch as long as they belong to the same
+repository.
 
 ## Explain Creating a Branch
 
-<Go into details of how to create branches from an existing repo>
+Branching allows us to branch out from the original code base and isolate our
+work from working code, whether it's on a solo project or a collaborative
+project.
+
+Changes in the branch that we branched from, or other branches, will not affect
+our branch without taking action to integrate other changes.
+
+To create a new branch from the branch we are currently on (in this case, it's
+the default `master` branch), we type the command `git branch new-branch-name`.
+
+If we want to create a new branch _and_ move onto that branch in a single
+command, we can use `git branch -b new-branch-name`.
+
+We can switch back to the previous branch using `git checkout -`, or to any
+branch that we're not currently on, we use `git checkout other-branch-name`.
+
```

Comment:
We might want to put the commands here in the format of code samples, to draw attention to them.